packages/next-app/src/app/stableswapsimulator/(components)/DepthCost.tsx

```diff
@@ -0,0 +1,213 @@
+"use client";
+
+import { bnum } from "@balancer-labs/sor";
+import { StableMath } from "@balancer-pool-metadata/math/src";
+import { PlotType } from "plotly.js";
+
+import Plot, { defaultAxisLayout } from "#/components/Plot";
+import { AnalysisData, useStableSwap } from "#/contexts/StableSwapContext";
+
+export default function DepthCost() {
+  const { indexAnalysisToken, baselineData, variantData } = useStableSwap();
+
+  const tokenY = baselineData?.tokens[indexAnalysisToken];
```

Comment:
hmmm I think we should keep the same convention for names and objects data to be consumed by the plot. If we decide this still we should refactor the other charts